doc/classes/DisplayServer.xml

```diff
@@ -1815,6 +1815,14 @@
 				[b]Warning:[/b] Advanced users only! Adding such a callback to a [Window] node will override its default implementation, which can introduce bugs.
 			</description>
 		</method>
+		<method name="window_start_drag">
+			<return type="void" />
+			<param index="0" name="window_id" type="int" default="0" />
+			<description>
+				Starts a window drag operation, using current mouse position. Call this method from a mouse-down event handler to simulate a mouse-down event on the window title bar. Using this method allows the window to participate in space switching, tiling and other system features.
```

Comment:
If you're using "mouse-down" often in DisplayServer keep that term for consistency (it's your reign here)
```suggestion
				Starts a drag operation on the window with the given [param window_id], using the current mouse position. Call this method when handling a mouse button being pressed to simulate a pressed event on the window's title bar. Using this method allows the window to participate in space switching, tiling and other system features.
```